app/src/components/common/icons/IconClose.tsx

```diff
@@ -12,18 +12,18 @@ const Wrapper = styled.svg<{ hoverEffect: boolean | undefined }>`
 
   .path {
     transition: 0.2s fill;
+    fill: ${props => props.theme.colors.tertiary};
```

Comment:
Ughhh man Im gonna kill you when I meet you in france this monday for real! So I tried explaining this 10 times but just change this line of code to this
`fill: ${props => (props.hoverEffect ? props.theme.colors.tertiary : props.theme.colors.textColorDark)};`
Try to understand why is that so or reread my comments and private chat....

Reply:
and If you compare create market button icon on master and your pr you woud have noticed they are not the same...